samcli/lib/deploy/deployer.py

```diff
@@ -401,7 +401,8 @@ def wait_for_execute(self, stack_name, changeset_type):
             raise deploy_exceptions.DeployFailedError(stack_name=stack_name, msg=str(ex))
 
         outputs = self.get_stack_outputs(stack_name=stack_name, echo=False)
-        self._stack_outputs(outputs)
+        if outputs:
+            self._stack_outputs(outputs)
```

Comment:
should this be given a more descriptive name? When I saw this code in your old PR, I thought it was functional - turns out this is just cosmetic.

Reply:
yeah true, I'll change it to display stack outputs.